package/environments/base.js

```diff
@@ -16,7 +16,7 @@ const getEntryObject = () => {
   const entries = {}
   const rootPath = join(config.source_path, config.source_entry_path)
 
-  globSync(`${rootPath}/**/*.*`).forEach((path) => {
+  globSync(`${rootPath}/*.*`).forEach((path) => {
```

Comment:
⚠️ This change is huge for those who choose to upgrade to webpacker 6, leave their pack file in subdirectories and didn't want to change `source_entry_path` from `packs` to `/`

In that case,
// This behavior retained
app/javascript/packs/application.js

// Missing pack file
app/javascript/packs/foo/fooPack.js
app/javascript/packs/foo/fooPack.scss